### PR TITLE
removed user locked field and used contact disabled field instead

### LINKED ISSUE #1763: Locking user not possible
Currently the permission tab doesn't have the locked toggle in the UI, this toggle is only visible in the contact tabs. Also, this toggle is only written to the contacts, and the fields in the user table are not used at all. Actually it would make more sense to lock the user, and remove this field from the contacts table.

### PR DESCRIPTION
**Pull request by danrot**
Should be merged with https://github.com/sulu-io/sulu-standard/pull/560

__tasks:__

- [x] test coverage
- [x] gather feedback for my changes
- [x] add locked toggler to permission tab
- [x] add disabled checkbox to details tab

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | fixes #1763
| BC breaks        | removed locked field in user entity
| Documentation PR | none

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## dev-develop
 
+### User locking
+
+The field `locked` from the user entity was removed. If you have set this field in your application,
+you have to move the values to the `disabled` field of the contact.
```
Comment (danrot) [inline UPGRADE.md]: Add a SQL, which achieves that.

**Comment by chirimoya** [comment]
Check `refreshUser`

**Comment by chirimoya** [comment]
ack

**Comment by chirimoya** [comment]
remove disabled from contact

**Comment by chirimoya** [comment]
Missing Upgrade

**Pull request merged**